doc/source/whatsnew/v1.2.0.rst

```diff
@@ -387,6 +387,11 @@ I/O
 - Removed ``private_key`` and ``verbose`` from :func:`read_gbq` as they are no longer supported in ``pandas-gbq`` (:issue:`34654`, :issue:`30200`)
 - Bumped minimum pytables version to 3.5.1 to avoid a ``ValueError`` in :meth:`read_hdf` (:issue:`24839`)
 
+Period
+^^^^^^
+
+- Bug in :meth:`DataFrame.replace` and :meth:`Series.replace` where :class:`Period` dtypes would be converted to object dytpes (:issue:34871)
```

Comment:
typo on dtypes

Reply:
> dytpes

typo is still there.  BTW in general let the person who made the original comment hit the "conversation resolved" button

Reply:
Gotcha thanks for the heads up, haven't used github at all really. Fixed